packages/cosmos_ui_components/lib/components/template/cosmos_balance_heading.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+class BalanceHeading extends StatelessWidget {
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 4, horizontal: 16),
```

Comment:
we can replace all those magic numbers in this file with `CosmosAppTheme.spacing*`

Reply:
Oops, apologies! I need to make it a habit now ⚡️

Reply:
✅